.github/workflows/test_couchdb.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+    paths:
+      - 'couchdb/**'
+  push:
+    paths:
+      - 'couchdb/**'
+
+jobs:
+  test:
+    name: Test CouchDB
```

Comment:
It could be added to the base `package.json` the same way as nginx/haproxy, though that will add up to 30s of extra execution time and in my opinion it's not worth that since it since it rarely changes. I'm not sure how required tests that are conditionally run will interact in github. There could be an argument to always run these tests on main branch push or release even if there's been no change to the couchdb directory though